redhat-cop/infra-ansible
# Nfs server

### What does this PR do?
Introduce the NFS server role to allow for an automated setup of NFS server and shares hosted by it

### How should this be tested?
Use the example in the `tests` directory and look at the `nfs-server` role README

**Note:** In order to test this in a cloud environment, for example OpenStack, make sure the proper security groups and rules have been added to the instance to successfully use the NFS server.

### Is there a relevant Issue open for this?
No

### Who would you like to review this?
cc: @etsauer @raffaelespazzoli @day4skiing
